### PR TITLE
media-libs/assimp: add 5.2.5

### PR DESCRIPTION
**Pull request by waebbl**
Bug: https://github.com/assimp/assimp/pull/4730
Closes: https://bugs.gentoo.org/869461
Signed-off-by: Bernd Waibel <waebbl-gentoo@posteo.net>

### CODE REVIEW

File: media-libs/assimp/assimp-5.2.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Importer library to import assets from 3D files"
+HOMEPAGE="https://github.com/assimp/assimp"
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/5.2.4"
```
Comment (juippis) [inline media-libs/assimp/assimp-5.2.5.ebuild:13]: Unfortunately it looks like this has to be bumped.
```
 FILES:-usr/lib64/libassimp.so.5.2.4
 FILES:+usr/lib64/libassimp.so.5.2.5
   ABI: libassimp.so.5(64) func(+105,-55) vars(+4) [BREAKING]
```

**Review by juippis — COMMENTED** [review]
I'm seeing 5.2.4-r1 installs a binary /usr/bin/assimp while 5.2.5 doesn't. Any idea where that comes from? And is missing the binary fatal?

File: media-libs/assimp/assimp-5.2.5.ebuild (hunk 1 above)
Comment (waebbl) [reply media-libs/assimp/assimp-5.2.5.ebuild:13]: Good catch. Must have overseen this.

**Comment by waebbl** [comment]
> I'm seeing 5.2.4-r1 installs a binary /usr/bin/assimp while 5.2.5 doesn't. Any idea where that comes from? And is missing the binary fatal?

Need to check on this. Not sure where it comes from or if its vital.

**Review by juippis — APPROVED** [review]
Thanks!

**Comment by waebbl** [comment]
Thank you!

**Pull request closed without merging**